agent/lib/kontena/rpc_server.rb

```diff
@@ -17,12 +17,12 @@ class RpcServer
     }
 
     class Error < StandardError
-      attr_accessor :code, :message, :backtrace
+      attr_reader :code
 
-      def initialize(code, message, backtrace = nil)
```

Comment:
The `RpcServer::Error` are only used locally, and do not need any `remote_backtrace`.